src/client.js

```diff
@@ -611,7 +611,14 @@ module.exports = class IrcClient extends EventEmitter {
     }
 
     setTopic(channel, newTopic) {
-        this.raw('TOPIC', channel, newTopic);
+        if (newTopic && newTopic.trim()) {
+            this.raw('TOPIC', channel, newTopic);
+            return;
+        }
+
+        // If newTopic is undefined or empty, remove the existing topic
+        // this is needed because without the : then it would be requesting the current topic
+        this.raw(`TOPIC ${channel} :`);
```

Comment:
Rather than multiplexing setTopic... can we just write the wrapper as clearTopic() or deleteTopic() instead?
Don't think it'd hurt much to add the name and would probably be easier to find than multiplexing given some funny input.

Reply:
I will add clearTopic() when I get a moment

Reply:
@brunnre8 happy with my change? I added a function but kept the multiplexing so that an empty string does not unexpectedly request the topic rather than set it

Reply:
Other than the need to specify the behavior in the docs it's fine for me yes ;)